docs/3.api/3.utils/prerender-routes.md

```diff
@@ -14,6 +14,10 @@ When prerendering, you can hint to Nitro to prerender additional paths, even if
 `prerenderRoutes` can only be called within the [Nuxt context](/docs/guide/going-further/nuxt-app#the-nuxt-context).
 ::
 
+::note
+`prerenderRoutes` has to be executed during prerendering. If the `prerenderRoutes` is used in dynamic pages/routes, then it will never be executed.
```

Comment:
```suggestion
`prerenderRoutes` has to be executed during prerendering. If the `prerenderRoutes` is used in dynamic pages/routes which are not prerendered, then it will not be executed.
```